.travis.yml

```diff
@@ -46,9 +46,12 @@ jobs:
           os: windows
           script:
             - choco install dotnetcore-sdk
+            - choco install dotnetfx
```

Comment:
It would be more informative when there's a failure if these were split into two separate builds:  `test on windows dotnetcore` and `test on windows dotnetfx`

Reply:
I've resolved this in a new commit